apps/docs/content/getting-started/react.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: React
+description: Getting started with Hopper UI.
+order: 1
+---
+
+This guide will help you getting started with Hopper UI. Hopper UI is a Design System that provides a collection of components, [tokens](/tokens), and [icons](/icons) to help you build applications.
+
+Below is the typical installation path, this assumes you are using React. If you have particluar needs you can have a look at these pages:
+
+- [Javascript installation](/getting-started/javascript)
+- [Individual package installation](/getting-started/individual-packages)
+
+## Installation
+
+Run the following command in your terminal to install the Hopper packages.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/styled-system react-aria react-aria-components" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
+import { HopperProvider } from "@hopper-ui/components";
+import { createRoot } from 'react-dom/client';
+
+const root = createRoot(document.getElementById('root'));
```

Comment:
you closed 2 of my comments but the 2 issues are still there: 
https://github.com/gsoft-inc/wl-hopper/pull/254#discussion_r1626269327
https://github.com/gsoft-inc/wl-hopper/pull/254#discussion_r1626565852